utils/storage/cookie.js

```diff
@@ -12,6 +12,7 @@ class CookieLocal {
   constructor(options) {
     this._options = {};
     this.options(options);
+    this.isSupportAvailable = this.isSupportAvailable();
```

Comment:
It's a bad idea to name both of them identical.

Reply:
Addressed